tab_downloads.md

```diff
@@ -9,11 +9,11 @@ tags: wstg
 
 ## Stable
 
-View the always-current stable version at [/stable](/stable).
+View the always-current stable version at [/stable](stable/).
 
-## [Unreleased 4.1] - 2020
+## [Version 4.1] - 2020-04-01
```

Comment:
```suggestion
## [Unreleased 5.0]

## [Version 4.1] - 2020-04-01
```

Reply:
Thanks! I’ll incorporate that when you we do the tag/release next week and I set the proper date in this PR.